ssl-impl/src/main/java/io/crate/protocols/ssl/SslConfiguration.java

```diff
@@ -84,7 +85,7 @@ static SslContext buildSslContext(Settings settings) {
                 trustManagers = trustStoreSettings.get().trustManagers;
             }
 
-            SSLContext sslContext = SSLContext.getInstance("TLS");
+            SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
I was thinking about opening a separate PR. I can do that if you want. Actually, it is not completely unrelated since we depend on a recent TLS version to support all browsers.